### PR TITLE
TH-574. Fix log messages.

### PR DESCRIPTION
**Pull request by miffn3**

### CODE REVIEW

File: src/main/java/com/exactpro/th2/act/ActHandler.java [hunk 1]
```diff
@@ -141,10 +141,10 @@ public void sendMessage(PlaceMessageRequest request, StreamObserver<SendMessageR
             responseObserver.onCompleted();
 
         } catch (RuntimeException | IOException e) {
-            logger.error("Send message failed. Message = {}", request.getMessage(), e);
+            logger.error("Failed to send a message. Message = {}", request.getMessage(), e);
             sendMessageErrorResponse(responseObserver, "Send message failed. See the logs.");
         } finally {
-            logger.debug("Send message finished during {}", System.currentTimeMillis() - startPlaceMessage);
+            logger.debug("Sending message has been finished in {}", System.currentTimeMillis() - startPlaceMessage);
```
Comment (jean-michel-perli-sonsin) [inline src/main/java/com/exactpro/th2/act/ActHandler.java:147]: Sending **the** message has been finished in

File: src/main/java/com/exactpro/th2/act/ActHandler.java [hunk 2]
```diff
@@ -247,7 +247,7 @@ private void placeMessage(PlaceMessageRequest request, StreamObserver<PlaceMessa
                         actName, System.currentTimeMillis() - startAwaitSync);
                 if (Context.current().isCancelled()) {
                     logger.warn("'{}' request cancelled by client", actName);
-                    sendErrorResponse(responseObserver, "Cancelled by client");
+                    sendErrorResponse(responseObserver, "Request has been cancelled by client");
```
Comment (jean-michel-perli-sonsin) [inline src/main/java/com/exactpro/th2/act/ActHandler.java:250]: **The** request has been cancelled by **the** client

File: src/main/java/com/exactpro/th2/act/ActHandler.java [hunk 3]
```diff
@@ -345,7 +345,7 @@ private boolean isSendPlaceMessage(PlaceMessageRequest request, StreamObserver<P
 
     private void sendMessage(PlaceMessageRequest request, EventID parentEventId) throws IOException {
         try {
-            logger.debug("Send message start");
+            logger.debug("Sending a message started");
```
Comment (jean-michel-perli-sonsin) [inline src/main/java/com/exactpro/th2/act/ActHandler.java:348]: Sending **the** message started

File: src/main/java/com/exactpro/th2/act/ActHandler.java [hunk 4]
```diff
@@ -362,7 +362,7 @@ private void sendMessage(PlaceMessageRequest request, EventID parentEventId) thr
                     .build();
             eventBatchMessageRouter.send(eventBatch, "publish", "event");
         } finally {
-            logger.debug("Send message end");
+            logger.debug("Sending a message ended");
```
Comment (jean-michel-perli-sonsin) [inline src/main/java/com/exactpro/th2/act/ActHandler.java:365]: Sending **the** message started

File: src/main/java/com/exactpro/th2/act/ActHandler.java [hunk 5]
```diff
@@ -469,16 +469,16 @@ private void sendMessageErrorResponse(StreamObserver<SendMessageResponse> respon
                         .build())
                 .build());
         responseObserver.onCompleted();
-        logger.debug("error response : {}", message);
+        logger.debug("Error response : {}", message);
     }
 
     private Checkpoint registerCheckPoint(EventID parentEventId) {
-        logger.debug("Register checkpoint start");
+        logger.debug("Registering a checkpoint start");
```
Comment (jean-michel-perli-sonsin) [inline src/main/java/com/exactpro/th2/act/ActHandler.java:476]: Registering **the** checkpoint **started**

File: src/main/java/com/exactpro/th2/act/ActHandler.java [hunk 6]
```diff
@@ -469,16 +469,16 @@ private void sendMessageErrorResponse(StreamObserver<SendMessageResponse> respon
...
         CheckpointResponse response = verifierConnector.createCheckpoint(CheckpointRequest.newBuilder()
                 .setParentEventId(parentEventId)
                 .build());
         if (logger.isDebugEnabled()) {
-            logger.debug("Register checkpoint end. Response " + shortDebugString(response));
+            logger.debug("Registering a checkpoint ended. Response " + shortDebugString(response));
```
Comment (jean-michel-perli-sonsin) [inline src/main/java/com/exactpro/th2/act/ActHandler.java:481]: Registering **the** checkpoint ended

**Review by jean-michel-perli-sonsin — CHANGES_REQUESTED** [review]

**Review by Nikita-Smirnov-Exactpro — APPROVED** [review]

**Pull request merged**